hosting/godot-html5-template/README.mdx

```diff
@@ -1,5 +1,9 @@
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
 # Godot HTML5 sample
 
+<MarkdownChipRow labels={["Sample project"]} />
```

Comment:
Isn't this label a bit redundant? The title already says that this is a sample (ditto for other pages).